cajita/src/Cajita_LocalMesh.hpp

```diff
@@ -120,6 +120,19 @@ class LocalMesh<Device, UniformMesh<Scalar>>
         return _ghost_high_corner[dim];
     }
 
+    // Get the extent of a given dimension.
+    KOKKOS_INLINE_FUNCTION
+    Scalar extent( Own, const int dim ) const
+    {
+        return _own_high_corner[dim] - _own_low_corner[dim];
```

Comment:
I would provide a templated version that calls `highCorner` and `lowCorner`

Reply:
You want a single function for both instantiations?

Reply:
```C++
template <typename OwnOrGhost>
KOKKOS_FUNCTION Scalar extent( OwnOrGhost w, const int dim ) const
{
       return highCorner(w, dim) - lowCorner(w, dim);
}
```

Reply:
oh yeah good call